extension-base/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "sample-http",
+  "version": "0.0.1"
```

Comment:
I would recommend releasing this as `1.0.0` right away. Otherwise semver matching become odd with tilde and caret matching.